neural_compressor/torch/algorithms/mix_precision/fp16_convert.py

```diff
@@ -0,0 +1,70 @@
+#
+# -*- coding: utf-8 -*-
+#
+# Copyright (c) 2024 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""FP16 Convert for Torch Modules."""
+
+from typing import Dict, Tuple
+
+import torch
+
+from neural_compressor.common import logger
+from neural_compressor.torch.algorithms.mix_precision.module_wrappers import FP16ModuleWrapper
+from neural_compressor.torch.quantization import MixPrecisionConfig
```

Comment:
Please do not import `xxx` from `neural_compressor.torch.quantization` into `neural_compressor.torch.algorithms`, it may cause the circular import.